lms/templates/footer.html

```diff
@@ -58,6 +58,7 @@
               <a href="${link['url']}">${link['title']}</a>
             </li>
           % endfor
+          <li><a href="https://www.edx.org/?utm_medium=affiliate_partner&amp;utm_source=opensource-partner&amp;utm_content=open-edx-partner-footer-link&amp;utm_campaign=open-edx-footer">${_("Take free online courses at edX.org")}</a></li>
```

Comment:
I wondered about that too, especially given how long the URL is, but you are right we don't want to make it a setting.  I could make it a constant in lms/djangoapps/branding/api.py.